PSKoans/Public/Get-Blank.ps1

```diff
@@ -20,11 +20,11 @@
         https://github.com/vexx32/PSKoans
     #>
     [CmdletBinding()]
-    [OutputType([void])]
+    [OutputType([Blank])]
     [Alias('__', 'FILL_ME_IN')]
     param()
 
     Write-Verbose "I AIN'T DOIN' NOTHIN'!!!"
 
-    $null
+    return New-Object -TypeName:Blank
```

Comment:
This function is called a _lot_ in the koans, and the additional overhead incurred with `New-Object` will be a bit of a drag on the overall performance when evaluating,
```suggestion
    [Blank]::new()
```

Reply:
Totally fair, was not aware of the performance overheard of New-Object, I'll start call ::new() from now on. Thanks for the tip!

And because I trust but verify I ran some numbers.

```powershell
Measure-Command  {
>>     for ($i = 0; $i -lt 10000; $i++) {
>>         [Blank]::New()
>>     }
>> }


Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 0
Milliseconds      : 76
Ticks             : 768778
TotalDays         : 8.89789351851852E-07
TotalHours        : 2.13549444444444E-05
TotalMinutes      : 0.00128129666666667
TotalSeconds      : 0.0768778
TotalMilliseconds : 76.8778


Measure-Command {
>>     for ($i = 0; $i -lt 10000; $i++) {
>>         New-Object -TypeName:Blank
>>     }
>> }


Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 1
Milliseconds      : 173
Ticks             : 11733668
TotalDays         : 1.35806342592593E-05
TotalHours        : 0.000325935222222222
TotalMinutes      : 0.0195561133333333
TotalSeconds      : 1.1733668
TotalMilliseconds : 1173.3668


```

Reply:
👍 wise policy!

In general, language features will always be faster than entire cmdlets; if there's a language feature that does it, use that instead. 😄